index.bs

```diff
@@ -8171,6 +8217,75 @@ The [=remote end steps=] are:
  1. Set the |authenticator|'s |isUserVerified| property to the |parameters|' |isUserVerified| property.
  1. Return [=success=].
 
+## <dfn>Set Credential Properties</dfn> ## {#sctn-automation-set-credential-properties}
```

Comment:
Should this be able to set `extensions` and `uvm` too?

Reply:
Ah right, what set of `extensions` the authenticator supports shouldn't change between assertions. But the `uvm` response can vary per request, not just per authenticator - for example if an authenticator supports both PIN and fingerprint UV, each assertion (even with the same credential) may report either method in the `uvm` response. So perhaps that should be variable per assertion, so you can test against that behaviour? But yeah, that should be a separate issue/PR in that case.